### PR TITLE
Fix topic info when schema is unavailable

### LINKED ISSUE #100: Display details for Connect dead letter topics
image

### LINKED ISSUE #111: No node details available for topics without schema
Error topics are not affected

Steps to reproduce:
- search for a topic and focus on it
- select another topic in the graph
- No node details available

### PR DESCRIPTION
**Pull request by disrupted**
Fixes #111
Fixes #100

### CODE REVIEW

File: backend/streams_explorer/core/services/schemaregistry.py [hunk 1]
```diff
@@ -11,10 +11,14 @@
 
 class SchemaRegistry:
     @staticmethod
-    def get_topic_value_schema_versions(topic: str) -> dict:
+    def get_topic_value_schema_versions(topic: str) -> list:
```
Comment (philipp94831) [inline backend/streams_explorer/core/services/schemaregistry.py:14]: was the typing beforehand wrong?

File: backend/tests/test_streams_explorer.py [hunk 2]
```diff
@@ -229,6 +223,24 @@ def test_get_node_information(self, streams_explorer, monkeypatch):
...
+                    name="test", value="test_value", type=NodeInfoType.BASIC
+                )
+            ],
+        )
+        assert streams_explorer.get_node_information(
+            "es-sink-connector-dead-letter-topic"
+        ) == NodeInformation(
+            node_id="es-sink-connector-dead-letter-topic",
+            node_type=NodeTypesEnum.ERROR_TOPIC,
+            info=[],
```
Comment (philipp94831) [inline backend/tests/test_streams_explorer.py:242]: can you also add a test please where this is not empty?

**Review by philipp94831 — CHANGES_REQUESTED** [review]

File: backend/streams_explorer/core/services/schemaregistry.py (hunk 1 above)
Comment (disrupted) [reply backend/streams_explorer/core/services/schemaregistry.py:14]: yes

File: backend/tests/test_streams_explorer.py (hunk 2 above)
Comment (disrupted) [reply backend/tests/test_streams_explorer.py]: I have added it now

Comment (philipp94831) [reply backend/tests/test_streams_explorer.py]: Why did you delete the old one though? It is always good to cover all cases

Comment (disrupted) [reply backend/tests/test_streams_explorer.py]: I'd say it's kinda non-standard behaviour for how we use the `LinkingService` but I have added clearing the nodeinfo and the empty check to the test now

**Review by philipp94831 — APPROVED** [review]
